wordpress-mobile/WordPress-Android
# Start blogging reminders from site settings

Fixes #14725 

This PR adds category to site settings that shows the current state of blogging reminders. This is either:
- No goals set
- N times a week

When the settings item is clicked, it opens the blogging reminders first screen. This will later be updated to skip the first screen. However, right now no other screen is available.

You can test the [fluxC changes](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2010) with this PR 

To test:
- Make sure the blogging reminders flag is turned off
- Go to site settings 
- Notice the "Blogging goals" item is missing
- Turn the flag on and restart the app
- Go to site settings
- Notice the "Blogging goal" item with the "None set" summary
- Click on it
- Notice the bottom sheet is opened

## Regression Notes
1. Potential unintended areas of impact
None

PR submission checklist:
